src/stories/Typography.js

```diff
@@ -40,10 +40,10 @@ storiesOf('Typography', module)
     return (
       <div>
         <div>
-          <Link to={`#`}>Internal Link</Link>
+          <Link href={`#`}>Internal Link</Link>
```

Comment:
Should this also be changed in the Navbar story as well?

Reply:
Yes definitely, thanks 🙏